### PR TITLE
Adds TGMC pack boxes

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Spend 100 points, get 150 to 200 points of value back in return, basically.
Some items (Mainly legendary stuff) can't be acquired anywhere else but here, like NVGs and PMTL rifles.

There's only four packs right now since my 1am raddled brain can't make any more up.

Weight list unchanged, 10% for legendary, 20% for Rare, 30 and 40 for uncommon and common respectively.

Legendaries are made to absolutely tear up the round if used well, rares much less so, being more so round spinners like two TL-102s or double howitzers and five mortars which will wack up the round but not really define it.

Uncommon and Commons are just made to be variety packs, giving you some random assortment of stuff.

image

## Why It's Good For The Game
Hopefully leads to more variety in what people use in rounds considering the amount of wacky stuff you'll see if RO's use it a decent amount, might need a cooldown and hopefully in the future will have more packs added as time goes on.

## Changelog
:cl:
add: Added TGMC Packs to Req, spend 100 points and get a curated random set of gear, with rarities to boot! Go open up some loot boxes. Now.
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
The intent is to provide marines with a sense of pride and accomplishment for unlocking different equipment.

As for cost, we selected initial values based upon data from the Order Quantities and other adjustments made to milestone boxes before introduction. Among other things, we're looking at average per-Marine point earn rates on a daily basis, and we'll be making constant adjustments to ensure that marines have challenges that are compelling, rewarding, and of course attainable via unga.

We appreciate the candid feedback, and the passion the corps has put forth around the current topics here on BlueSpaceHub, our forums and across numerous space media outlets.

Our team will continue to make changes and monitor marine feedback and update everyone as soon and as often as we can.

**Comment by Al-1ce** [comment]
ROs gambling addiction begins NOW.

**Comment by Oscari-pog** [comment]
thermobaric

**Comment by Araulius** [comment]
I would love to order this, just so I don't have to choose a kit to focus on. RNG kit choosing.

**Comment by Pariah919** [comment]
I expected far more feedback and I want to throw this out into the wild for a test run.

**Comment by Lantekk** [comment]
I'd like for some non-weapon packs, perhaps some with materials or sentries.
Or one that's just neuraline injectors.

**Comment by Araulius** [comment]
I got a banhammer in my loot crate, and it doesn't even bwoink when I hit people with it. Literally unplayable.

File: code/game/objects/items/loot_box.dm [hunk 1]
```diff
@@ -71,3 +71,247 @@
 		/obj/item/toy/sword,
 		/obj/item/weapon/banhammer,
 	)
+
+// 150 to 200 points of value packs, spend 100 points get 150 to 200 in value, basically. Ideally, commons are variety packs, uncommons maybe shake up the round a bit, rares a bit more. Legendaries make the round go wacko. You get a crate of stuff dropped on spawn.
+
+/obj/item/loot_box/tgmclootbox
+	name = "TGMC pack box"
+	desc = "A box of gear sent over by the TGMC on request, nobody knows whats in it. You just know it'll probably be good."
```
Comment (XSlayer300) [inline code/game/objects/items/loot_box.dm:79]:
```suggestion
	desc = "A box of gear sent over by the TGMC on request, nobody knows what's in it. You just know it'll probably be good."
```

**Review by Lumipharon — APPROVED** [review]
No one has ree'd about OP req buff, so seems alright

**Comment by Lumipharon** [comment]
> I misread the title when this got merged and I thought marines were getting little juice boxes with their MREs.

juice poppers when?

**Pull request merged**